src/util/ajax.js

```diff
@@ -148,16 +148,15 @@ function makeFetchRequest(requestParameters: RequestParameters, callback: Respon
             if (response.ok) {
                 const cacheableResponse = cacheIgnoringSearch ? response.clone() : null;
                 return finishRequest(response, cacheableResponse, requestTime);
-
             } else {
                 return callback(new AJAXError(response.statusText, response.status, requestParameters.url));
             }
         }).catch(error => {
-            if (error.code === 20) {
+            if (error.name === 'AbortError') {
```

Comment:
Nit: any reason for this change? Can an abort error be other than code 20, or is this just for clarity?

Reply:
Yep, it's just for the clarity